test/Api/DevCycleClientTest.php

```diff
@@ -199,16 +199,16 @@ public function testTrackNoType()
 
     public function testOpenFeature()
     {
-        $boolResult = self::$openFeatureClient->getBooleanValue('php-sdk', false, self::$context);
+        $boolResult = self::$openFeatureClient->getBooleanValue('test', false, self::$context);
         self::assertTrue($boolResult);
 
-        $numberResult = self::$openFeatureClient->getIntegerValue('php-sdk-integer', -1, self::$context);
-        self::assertEquals(1, $numberResult);
+        $numberResult = self::$openFeatureClient->getIntegerValue('test-number-variable', -1, self::$context);
+        self::assertNotEquals(-1, $numberResult);
 
-        $stringResult = self::$openFeatureClient->getStringValue('php-sdk-string', 'default', self::$context);
-        self::assertEquals('string', $stringResult);
+        $stringResult = self::$openFeatureClient->getStringValue('test-string-variable', 'default', self::$context);
+        self::assertNotEquals('default', $stringResult);
 
-        $structResult = self::$openFeatureClient->getObjectValue('php-sdk-struct', array(), self::$context);
-        self::assertEquals(array("key"=>"value", "number"=>1, "bool"=>true, "nested"=>array("key"=>"value")), $structResult);
+        $structResult = self::$openFeatureClient->getObjectValue('test-json-variable', array(), self::$context);
```

Comment:
should we not test this with an object response?

Reply:
Not really - we can't guarantee the format given the source test projects change all the time - this just prevents us from breaking things; I'm confident that the other tests provide safety on this (namely test harness)